jobs/integr8ly/msb-integration-test.yaml

```diff
@@ -0,0 +1,162 @@
+    project-type: pipeline
+    concurrent: false
+    properties:
+      - build-discarder:
+          num-to-keep: 15
+    parameters:
+      - string:
+          name: 'installationGitUrl'
+          default: 'https://github.com/integr8ly/installation.git'
+          description: '[REQUIRED] Integreatly installer Git URL'
+      - string:
+          name: 'installationGitBranch'
+          default: 'master'
+          description: '[REQUIRED] The name of the git branch to be used for installing Integreatly'
+    dsl: |
+        def testOptions = [:]
+        final QE_REPO_URL = 'https://gitlab.cee.redhat.com/integreatly-qe/integreatly-qe.git'
+
+        def verifyParameters(testOptions) {
+          if (!testOptions.gitUrl || !testOptions.gitBranch) {
+            def userInput = input message: 'Integreatly Test Options', parameters: [
+              string(defaultValue: (testOptions.gitUrl ?: 'https://github.com/integr8ly/installation.git'), description: 'Integreatly installer Git URL', name: 'installationGitUrl'),
+              string(defaultValue: (testOptions.gitBranch ?: 'master'), description: 'Integreatly Git Branch', name: 'installationGitBranch'),
+            ]
+
+            testOptions.gitUrl = userInput.installationGitUrl
+            testOptions.gitBranch = userInput.installationGitBranch
+            testOptions.clusterName = userInput.installationGitBranch
+            testOptions.openshiftMasterUrl = "https://${userInput.installationGitBranch}.${testOptions.domainName}"
+            verifyParameters(testOptions)
+          }
+        }
+
+        def setClusterAdminCredentials() {
+          def integreatlyCredentialsID = 'tower-openshift-cluster-credentials'
+          def clusterAdminCredentials = [:]
+          withCredentials([usernamePassword(credentialsId: integreatlyCredentialsID, usernameVariable: 'CLUSTER_ADMIN_USERNAME', passwordVariable: 'CLUSTER_ADMIN_PASSWORD')]) {
+            clusterAdminCredentials.clusterAdminUsername = "${CLUSTER_ADMIN_USERNAME}"
+            clusterAdminCredentials.clusterAdminPassword = "${CLUSTER_ADMIN_PASSWORD}"
+          }
+          return clusterAdminCredentials
+        }
+
+        def clusterAdminCredentials = setClusterAdminCredentials()
+
+        try {
+          timeout(180) { ansiColor('gnome-terminal') { timestamps {
+            node('cirhos_rhel7') {
+              
+              stage('Verify Parameters') {
+                testOptions.awsAccountName = 'fheng.AWS'
+                testOptions.awsRegion = 'eu-west-2'
+                testOptions.clusterName = params.installationGitBranch
+                testOptions.domainName = 'skunkhenry.com'
+                testOptions.gitUrl = params.installationGitUrl
+                testOptions.gitBranch = params.installationGitBranch
+                testOptions.openshiftMasterUrl = "https://${params.installationGitBranch}.${testOptions.domainName}"
+                verifyParameters(testOptions)
+              }   
```

Comment:
We can remove this stage in the future or just change it in default params initialization.